Cargo.toml

```diff
@@ -648,7 +662,25 @@ snmalloc-rs = { version = "0.3.7", features = ["build_cc"] }
 # See: https://github.com/eira-fransham/crunchy/issues/13
 crunchy = "=0.2.2"
 
-# [patch.crates-io]
+[patch.crates-io]
+alloy-evm = { git = "https://github.com/alloy-rs/evm", rev = "beb6832" }
+alloy-op-evm = { git = "https://github.com/alloy-rs/evm", rev = "beb6832" }
+
+revm = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-bytecode = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-database = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-state = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-primitives = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-interpreter = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-inspector = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-context = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-context-interface = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-database-interface = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-specification = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+revm-optimism = { git = "https://github.com/bluealloy/revm", rev = "a8a9893b" }
+
+revm-inspectors = { git = "https://github.com/paradigmxyz/revm-inspectors", rev = "8900c2b" }
```

Comment:
Why do we need these patches here?

Reply:
haven't looked into it, but this is patched upstream.
https://github.com/paradigmxyz/reth/blob/main/Cargo.toml#L679